app/models/article-model.js

```diff
@@ -15,4 +15,18 @@ exports.fetchArticlesArr = () => {
     .then((response) => {
         return response.rows
     })
+}
+
+exports.addVotes = (id, num) => {
+    return db.query(`SELECT * FROM articles WHERE article_id = ${id}`)
+    .then((result) => {
+        if(result.rowCount === 0){
+            return Promise.reject({status:404, msg:'id not found'})
+        }else if(typeof num !== 'number'){
+            return Promise.reject({status: 400, msg: 'Missing Required Fields'})
+        }
+        const article = result.rows[0]
+        article.votes += num
```

Comment:
This should be done within your PSQL. A PSQL error would also then be thrown if num wasn't a number (as it breaks the constraints) Have a look at how you can update a value via SQL